webapp/src/components/Vendor/decentraland/NFTTopbar/SelectedFilters/SelectedFilters.tsx

```diff
@@ -0,0 +1,118 @@
+import { useState, useMemo } from 'react'
+import { Rarity } from '@dcl/schemas'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+import { useCallback, useEffect } from 'react'
+import { Pill } from './Pill/Pill'
+import { Props } from './SelectedFilters.types'
+import styles from './SelectedFilters.module.css'
+import { getCollectionNameByAddress } from './utils'
+import { Mana } from 'decentraland-ui'
+import classNames from 'classnames'
+
+export const SelectedFilters = ({ browseOptions, isLandSection, onBrowse }: Props)  => {
+  const { rarities, network, onlySmart, contracts, wearableGenders, onlyOnSale, emotePlayMode, minPrice, maxPrice, onlyOnRent } = browseOptions;
+  const [collection, setCollection] = useState<string | undefined>(undefined)
```

Comment:
```suggestion
  const [collection, setCollection] = useState<string | undefined>()
```
I believe the `undefined` is redundant here, it will be the default value if not specified